hypertrace-graphql-labels-schema-api/src/main/java/org/hypertrace/graphql/label/schema/rule/StringCondition.java

```diff
@@ -3,28 +3,51 @@
 import graphql.annotations.annotationTypes.GraphQLField;
 import graphql.annotations.annotationTypes.GraphQLName;
 import graphql.annotations.annotationTypes.GraphQLNonNull;
+import java.util.List;
+import javax.annotation.Nullable;
 
 @GraphQLName(StringCondition.TYPE_NAME)
 public interface StringCondition {
   String TYPE_NAME = "LabelApplicationStringCondition";
 
   String OPERATOR_KEY = "operator";
   String VALUE_KEY = "value";
+  String VALUES_KEY = "values";
+  String STRING_CONDITION_VALUE_TYPE_KEY = "stringConditionValueType";
 
   @GraphQLName(Operator.TYPE_NAME)
   enum Operator {
     OPERATOR_EQUALS,
-    OPERATOR_MATCHES_REGEX;
+    OPERATOR_MATCHES_REGEX,
+    OPERATOR_MATCHES_IPS,
+    OPERATOR_NOT_MATCHES_IPS;
     private static final String TYPE_NAME = "StringConditionOperator";
   }
 
+  @GraphQLName(StringConditionValueType.TYPE_NAME)
+  enum StringConditionValueType {
+    VALUE,
+    VALUES;
+    private static final String TYPE_NAME = "StringConditionValueType";
+  }
+
   @GraphQLField
   @GraphQLNonNull
   @GraphQLName(OPERATOR_KEY)
   Operator operator();
 
   @GraphQLField
-  @GraphQLNonNull
+  @Nullable
```

Comment:
Another alternative would be to have value be typed Object and accept either a string or list of strings in it. We're not consistent, we've taken both approaches in different places. Up to you

Reply:
If no harm in this approach, will keep it this way.